map/src/main/kotlin/edu/artic/map/rendering/GlideMapTileProvider.kt

```diff
@@ -0,0 +1,43 @@
+package edu.artic.map.rendering
+
+import android.content.Context
+import com.bumptech.glide.Glide
+import com.google.android.gms.maps.model.Tile
+import timber.log.Timber
+import java.io.IOException
+import kotlin.math.pow
+
+/**
+ * Description:
+ */
+class GlideMapTileProvider(private val context: Context,
```

Comment:
This is the `TileProvider` we'll be using, correct?

Reply:
This is the tile provider we will be using. it is also the tile provider that will be updated to use a dynamic url